.redhat-ci.yml

```diff
@@ -0,0 +1,15 @@
+host:
+    distro: fedora/24/cloud
```

Comment:
Sure, that's fine. But in reality, you'll need both here. You can't containerize everything when what you're testing is containerization itself. Projects like atomic, docker-storage-setup, and rpm-ostree's vmcheck suite can't run *in* Docker, they need to manipulate Docker and/or the host itself.

But otherwise I agree, projects which *can* run in Docker should just use AH. I'm hoping to add some more sugar to the spec to make this easier and more efficient than provisioning a VM. But for now, they'll have to start and run Docker manually in the `tests` section.